pulse2percept/datasets/tests/test_nanduri2012.py

```diff
@@ -11,23 +11,23 @@ def test_load_nanduri2012():
     columns = ['subject', 'implant', 'electrode', 'task', 'stim_class',
                'freq', 'amp_factor', 'ref_stim_class', 'ref_amp_factor',
                'ref_freq', 'brightness', 'size', 'pulse_dur',
-               'interphase_dur', 'pulse_type', 'source']
+               'interphase_dur', 'pulse_type', 'varied_param']
```

Comment:
Please explain this new feature in https://github.com/pulse2percept/pulse2percept/blob/master/pulse2percept/datasets/nanduri2012.py#L40